libcxx/docs/UserDocumentation.rst

```diff
@@ -94,6 +94,24 @@ enable or disable extended libc++ behavior.
   only intended to be used by vendors and changing their value from the one provided
   in your toolchain can lead to unexpected behavior.
 
+**_LIBCPP_DISABLE_DEPRECATION_WARNINGS**:
```

Comment:
Can you please confirm that in this whole section you only moved things around and didn't make any drive-by changes?